### PR TITLE
Push from feature/fix-dependencies

### PR DESCRIPTION
**Pull request by joaquinmarti**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-  "version": "0.0.66",
+  "version": "0.0.67",
   "description": "",
   "license": "ISC",
   "type": "module",
   "homepage": "https://backlight.dev/preview/wzPn6NO54tDJlBxCtjGx/",
   "scripts": {
     "serve": "npx backlight@latest serve wzPn6NO54tDJlBxCtjGx --open"
+  },
+  "devDependencies": {
```
Comment (jorenbroekema) [inline package.json:11]: See my comment in discord about devDependencies/dependencies distinction

Comment (bashmish) [reply package.json:11]: nitpick: put devDependencies below dependencies

File: package.json [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
+    "lodash": "latest",
+    "path": "latest",
+    "prism-react-renderer": "latest",
+    "prop-types": "latest",
+    "react": "latest",
+    "react-dom": "latest",
+    "react-live": "latest",
+    "styled-components": "latest",
+    "styled-system": "latest",
+    "tailwindcss": "latest"
```
Comment (bashmish) [inline package.json:28]: you shouldn't use `latest` here, I should have been more clear in Discord

you should pin the latest, meaning you find which actual number is latest and pin `^x.y.z`

File: package.json [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+    "react": "latest",
+    "react-dom": "latest",
+    "react-live": "latest",
+    "styled-components": "latest",
+    "styled-system": "latest",
+    "tailwindcss": "latest",
+    "twind": "latest"
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (gqio) [reply package.json:30]: I think the issue comes the double formatter we have. I didn't dig it much into it.
Save all fix correctly the files

File: package.json (hunk 2 above)
Comment (joaquinmarti) [reply package.json]: Got it, that's what I wasn't understanding regarding the version. Thanks!

**Review by bashmish — APPROVED** [review]

File: tailwind-showcases/src/TailwindShowcases.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import 'twind/shim';
```
Comment (gqio) [inline tailwind-showcases/src/TailwindShowcases.tsx:2]: I thought Georges said it's not ok here.

**Review by gqio — CHANGES_REQUESTED** [review]

File: tailwind-showcases/src/TailwindShowcases.tsx (hunk 4 above)
Comment (joaquinmarti) [reply tailwind-showcases/src/TailwindShowcases.tsx]: Done @gqio

**Review by gqio — APPROVED** [review]

**Pull request merged**